README.md

```diff
@@ -31,4 +34,62 @@ console.log(v1.minus(v2));
 var v3 = new Vector(3, 4)
 console.log(v3.getLength());
 // => 5
-```
+``` -->
+
+
+1.
+class Human {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+
+  ageOneYear() {
+    this.age++
+    return `You are now ${this.age} years old`;
+  }
+
+  eating() {
+    return `mmm, mmm, mmm, I'm love'n it`
+  }
+}
+
```

Comment:
The eating function should console.log that string, not return it. When a question asks you to log something, it usually means to console.log it.